### PR TITLE
[CIFuzz] Add cifuzz- prefix to all artifacts

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Do this to avoid conflicts with artifacts uploaded by actions other than cifuzz.

### CODE REVIEW

**Review by inferno-chromium — APPROVED** [review]

**Review by oliverchang — COMMENTED** [review]
LGTM

File: infra/cifuzz/filestore/github_actions/__init__.py [hunk 1]
```diff
@@ -46,12 +46,20 @@ class GithubActionsFilestore(filestore.BaseFilestore):
...
     self.github_api_http_headers = github_api.get_http_auth_headers(config)
 
+  def _get_artifact_name(self, name):
+    if name.startswith(self.ARTIFACT_PREFIX):
+      return name
+    return f'{self.ARTIFACT_PREFIX}{name}'
+
   def upload_directory(self, name, directory):  # pylint: disable=no-self-use
     """Uploads |directory| as artifact with |name|."""
+    name = self.get_artifact_name(name)
```
Comment (oliverchang) [inline infra/cifuzz/filestore/github_actions/__init__.py:62]: FYI this is fixed in https://github.com/google/oss-fuzz/pull/6087.

**Review by oliverchang — COMMENTED** [review]
Also, do we need a unique artifact name for crashes after every run?

File: infra/cifuzz/filestore/github_actions/__init__.py (hunk 1 above)
Comment (jonathanmetzman) [reply infra/cifuzz/filestore/github_actions/__init__.py:62]: Thanks!

**Comment by jonathanmetzman** [comment]
> Also, do we need a unique artifact name for crashes after every run?

Why does it need to be unique? Uploading artifacts with the same name doesn't overwrite old ones.

**Pull request merged**